starterFiles/components/avatar.css

```diff
@@ -0,0 +1,34 @@
+@import url('../abstracts/variables.css');
+@import url('../base/reset.css');
+@import url('../utils/util.css');
+@import url('../utils/margin.css');
+@import url('../utils/padding.css');
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
```

Comment:
Don't need fonts for the component library as per the feedback I got yesterday.
You can add fonts to your documentation site when you make it.